public/js/flashMessage.js

```diff
@@ -1,18 +1,18 @@
-window.schuldhulp = window.schuldhulp || {};
-
-window.schuldhulp.flashMessage = function (flashDom) {
-    window.setTimeout(function () {
-        flashDom.remove();
-    }, 9000);
-    flashDom.querySelector('a[href="#close-flash"]').addEventListener('click', (function (event) {
-        flashDom.remove();
-        event.preventDefault();
-    }));
-};
-
-document.addEventListener('DOMContentLoaded', function(event) {
-    var flashDoms = document.getElementsByClassName('flash-message');
-    for (var i = 0; i < flashDoms.length; i ++) {
-        new window.schuldhulp.flashMessage(flashDoms.item(i));
-    }
-});
+window.schuldhulp = window.schuldhulp || {};
```

Comment:
Hier zijn de codewijzigingen niet leesbaar. Als er geen codewijzigingen zijn, moet dit stuk er uit of anders alleen de diffs tonen.